ambianic-debug.sh

```diff
@@ -27,6 +27,8 @@ docker pull ambianic/ambianic-edge:dev
 docker run -it --rm \
   --name ambianic-edge-dev \
   --mount type=bind,source="$MY_DIR",target=/workspace \
+  -v /opt/vc:/opt/vc \
```

Comment:
Would /opt/vc work on a non-raspberry pi machine? Do we need to add conditional logic in the shell file for this parameter?

Reply:
yes you are right, will provide it